exhibits-solr-conf.gemspec

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'exhibits-solr-conf/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'exhibits-solr-conf'
```

Comment:
I have to admit as somebody who has created my fair share of gems with hyphens\ I eventually end up regretting it (saved key-strokes aside).

I have been noticing other gems even strongly enforcing switching to the underscore version (outputting strong warning messages when installing the hyphenated version, etc).

The hyphens vs. underscores do have semantic differences (as much as that irritates me).